src/error/specification

```diff
@@ -1,1354 +1,48 @@
-number:0
-description:a test for an exception specification
+number:C00000
+description:A test for an exception specification
 :new lines to the same entry can be started with a colon
-:the number is sequential and has no special meaning
+:the code is similar to SQLSTATES and corresponds
+:to a specific category
 :
 :there are no comments possible in this file
-ingroup:test
-:a unique identifier for a doxygen module
-module:test
-:the libloader's or plugin's module name
-class:Ok
-:a class name which can be used by bindings
-:they might add Pre- or Postfixes though
 ignore:true
 :a generator should always ignore this first entry
 :or others which have ignore:true.
 :It is for errors which are not relevant anymore
 :an empty line will start the next error specification
 
-number:1
-ingroup:modules
-description:could not load module, dlopen failed
-module:dl
-severity:warning
-class:FailedLoadModuleError
-extends:ModuleError
+number:C01100
+description:Resource
```

Comment:
This description does not help.

Reply:
What exactly do you mean by "does not help"? Should I write something like `Resource related` or a whole label list picked from the categorization guideline, eg. `False permissions, missing files/ directory, not enough space, resource, etc.`? The description here is just a very short headline of the underlying error

Reply:
IMO a link to the decision document containing the full descriptions should be good enough.

Reply:
This is exactly what I have thought of. I added the link to the top of the file

Reply:
Ok, good solution!

Do we need this "description:" in this file then at all?

Reply:
Still open: why would we need "description:" at all here?